index.html

```diff
@@ -5,19 +5,70 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nerko+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Heebo:wght@600&display=swap"
+      rel="stylesheet"
+    />
     <title>Awsome Books</title>
   </head>
   <body>
+    <nav class="nav-bg">
+      <div class="title">
+        <h2 class="home">Awsome Books</h2>
+      </div>
+      <div class="nav-list">
+        <ul>
+          <li class="Home"><a href="#">list</a></li>
+          <li><div class="line"></div></li>
+          <li class="add-new"><a href="#">Add new</a></li>
+          <li><div class="line"></div></li>
+          <li class="contact-link"><a href="#">contact</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <div></div>
+
     <div class="Books-Container">
-      <h1>AWSOME BOOKS</h1>
-      <div class="books-list"></div>
+      <div class="awesome-books">
+        <h1>AWSOME BOOKS</h1>
+        <div class="books-list"></div>
+      </div>
+      <div class="new-book">
+        <h1>Add a new book</h1>
+        <form>
+          <input type="text" id="Title" placeholder="Title" /><br /><br />
+          <input type="text" id="Author" placeholder="Author" /><br /><br />
+          <input type="button" id="Add-Button" value="Add" />
+        </form>
+      </div>
+      <div class="contact">
+        <div class="info">
+          <h1>Contact information</h1>
```

Comment:
- [optional] For SEO optimization, it is always better to have one `h1` tag in an HTML document. Kindly change this to other header tag aside `h1`. Thanks. ℹ️

Reply:
I WILL LOOK INTO THAT